packages/react-components/react-slider/stories/src/Slider/SliderControlled.stories.tsx

```diff
@@ -18,7 +18,9 @@ export const Controlled = () => {
         onChange={onSliderChange}
         id={id}
       />
-      <Button onClick={resetSlider}>Reset</Button>
+      <Button onClick={resetSlider} disabledFocusable={sliderValue === 0}>
```

Comment:
seems to be unrelated, but let's keep it 🙂

Reply:
ooooops haha, that was me doing a quick experiment for an idea for a different PR/a11y fix :D

I'll split it out lol, thanks for catching that